frontend/src/metabase/public/components/EmbedModal/EmbedModalContent/EmbedModalContent.tsx

```diff
@@ -118,14 +118,33 @@ export const EmbedModalContent = (
         parameterValues[parameter.id] ?? null,
       ]),
     );
-  };
+  }, [embeddingParams, parameterValues, resourceParameters]);
 
-  const previewParametersBySlug = getPreviewParamsBySlug();
   const lockedParameters = getLockedPreviewParameters(
     resourceParameters,
     embeddingParams,
   );
 
+  const iframeUrl = useMemo(() => {
```

Comment:
This fixes the problem where the iframe is rendered twice.

https://www.loom.com/share/c082515a29724bcdb7d8a67535f21a1c